docs/api/ref/Events.md

```diff
@@ -2,30 +2,61 @@
 title: Events
 ---
 
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
 In addition to intent and context events, the FDC3 API and PrivateChannel API may be used to listen for other types of events via their `addEventListener()` functions.
 
 ## `ApiEvent`
 
 Type defining a basic event object that may be emitted by an FDC3 API interface such as DesktopAgent or PrivateChannel. There are more specific event types defined for each interface.
 
+<Tabs groupId="lang">
+<TabItem value="ts" label="TypeScript/JavaScript">
+
 ```ts
 interface ApiEvent {
   readonly type: string;
   readonly details: any;
 }
 ```
 
+</TabItem>
+<TabItem value="golang" label="Go">
+
+```go
+type ApiEvent struct {
+  Type string 
+  Details any
+}
+```
+
+</TabItem>
+</Tabs>
```

Comment:
```suggestion
</Tabs>

```
Needs a clear line under this as the next heading is not rendering:
![image](https://github.com/user-attachments/assets/f77f04dd-7381-468f-acae-b9131166c1a2)